components/de_amp/browser/test/de_amp_browsertest.cc

```diff
@@ -53,7 +58,18 @@ const char kTestNonAmpBody[] =
 class DeAmpBrowserTest : public InProcessBrowserTest {
  public:
   DeAmpBrowserTest() {
-    feature_list_.InitAndEnableFeature(de_amp::features::kBraveDeAMP);
+    std::vector<base::Feature> disabled_features = {};
+#if BUILDFLAG(IS_MAC)
+    // On Mac, the DeAmpBrowserTest.AmpURLNotStoredInHistory test crashes
+    // due to https://crbug.com/1284500: DCHECK in
+    // blink::ContentToVisibleTimeReporter::TabWasShown when BFCache is
+    // used. To get around the crash, disabling BFCache for these tests
+    // until the upstream bug is fixed.
+    disabled_features.push_back({features::kBackForwardCache});
```

Comment:
how are we tracking this so we know when we can remove this?

Reply:
I'll an issue to track this.

Reply:
Created https://github.com/brave/brave-browser/issues/22347